coffee_cmd/src/coffee_term/command_show.rs

```diff
@@ -22,6 +22,7 @@ pub fn show_list(coffee_list: Result<CoffeeList, CoffeeError>) -> Result<(), Cof
         term::format::bold(String::from("Language")),
         term::format::bold(String::from("Name")),
         term::format::bold(String::from("Enabled")),
+        term::format::bold(String::from("Git HEAD")),
```

Comment:
a user need also a git PHD to understand what is a git HEAD? Idk if this is necessary.

Reply:
We already print the git HEAD to the user with `coffee remote list`
If we add the possibility to manage the branch for the plugin in separation from the repository, I think we have to print it in `coffee list`

Reply:
Sure. Do you want me to rename "Git HEAD" to "commit" in `coffee remote list` output too?